### DISCUSSION TITLE
Signed Library

### DISCUSSION THREAD

**Discussion by snehgin** [QuestPDF/QuestPDF#471] [Q&A] [answered]
I could see that the Nuget is not Signed or doesn't have a strong name. Is it possible to get a signed version of the DLLs?

**Comment by girlpunk** [comment]
I believe this was requested in #461, however no progress as of yet.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

The QuestPDF NuGet package is now strong-named and signed since version 2024.10.3.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
